packages/vue-flicking/src/Flicking.ts

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) 2015 NAVER Corp.
+ * egjs projects are licensed under the MIT license
+ */
+
+import NativeFlicking, { Plugin, FlickingOptions, withFlickingMethods, DEFAULT_OPTIONS } from "@egjs/flicking";
+import ChildrenDiffer from "@egjs/vue-children-differ";
+import ListDiffer, { DiffResult } from "@egjs/list-differ";
+import { Component, Vue, Prop } from "vue-property-decorator";
+import { CreateElement, VNodeData, VNode } from "vue";
+
+@Component({
+  name: "Flicking",
+  directives: {
+    "children-differ": ChildrenDiffer,
+  },
+})
+export default class Flicking extends Vue {
+  // Tag of wrapper element
+  @Prop({ type: String, default: "div", required: false }) tag!: string;
+  @Prop({ type: Object, default: () => ({}), required: false }) options!: Partial<FlickingOptions>;
+  @Prop({ type: Array, default: () => ([]), required: false }) plugins!: Plugin[];
+
+  // Following decorator will inject native Flicking's method into Vue-Flicking
+  @withFlickingMethods
+  private $_nativeFlicking!: NativeFlicking;
+  private $_pluginsDiffer!: ListDiffer<Plugin>;
+  private $_cloneCount!: number;
+
+  public mounted() {
+    this.$_pluginsDiffer = new ListDiffer<Plugin>();
+    this.$_cloneCount = 0;
+
+    const options = {...this.options, ...{ renderExternal: true }};
+    this.$_nativeFlicking = new NativeFlicking(this.$el as HTMLElement, options);
+
+    this.$_bindEvents();
+    this.$_checkUpdate();
+  }
+
+  public render(h: CreateElement) {
+    const classPrefix = this.options.classPrefix || DEFAULT_OPTIONS.classPrefix;
+    const viewportData: VNodeData = {
+      class: {},
+    };
+    const cameraData: VNodeData = {
+      class: {},
+      directives: [{ name: "children-differ", value: this.onUpdate.bind(this) }],
+    };
+    viewportData.class[`${classPrefix}-viewport`] = true;
+    cameraData.class[`${classPrefix}-camera`] = true;
+
+    const panels = this.$_getPanels();
+
+    return h(this.tag,
+      [h("div", viewportData,
+        [h("div", cameraData,
+          panels,
+        )],
+      )],
+    );
+  }
+
+  public onUpdate(diffResult: DiffResult<HTMLElement>) {
+    this.$_nativeFlicking.sync(diffResult as DiffResult<HTMLElement>);
```

Comment:
Already the diffResult is DIffResult <HTMLElement> type.

Reply:
That's my mistake, I'll fix it